app/helpers/signs_helper.rb

```diff
@@ -17,7 +17,7 @@ def render_grammar_notes(sign)
                 'Sign', 'Click', 'glossary #{note}']);" }
       link_to(t("signs.show.field.#{note}"),
               "#{Page.find(Setting.get(:glossary)).try(:path)}##{note}", attrs)
-    end.compact.join(', ').html_safe
+    end.compact.join(', ').html_safe # rubocop:disable Rails/OutputSafety
```

Comment:
gosh, there are a lot of these.

I would rather leave them as failing cops, then they appear on a list to be fixed, instead of making them to pass silently.